packages/rn-tester-e2e/tests/specs/components/button/cancelButton.test.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+const {ComponentsScreen} = require('../../../screens/components.screen.js');
+const {
+  ButtonComponentScreen,
+} = require('../../../screens/components/buttonComponent.screen.js');
+
+const cancelText = 'Your application has been cancelled!';
+
+describe('Test is checking cancel button', function () {
```

Comment:
Nit (optional recommendation, up to you): `describe()` should explain the feature as a whole being tested. Thinking about it from improving the debugging experience, I think the description should be:

"Testing Cancel Button Functionality"

This tells the engineer exactly which feature is under test.